### PR TITLE
Feature/nutr home page

### PR DESCRIPTION
**Pull request by buffginger**
Check discord for picture of the home page. 
I still need to edit the VirtualTable to take in table data as props, right now it has dummy data.

### CODE REVIEW

File: zoo_frontend/src/getPageContext.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { SheetsRegistry } from 'jss';
 import { createMuiTheme, createGenerateClassName } from '@material-ui/core/styles';
-import * as Colors from '@material-ui/core/colors';
+// import * as Colors from '@material-ui/core/colors';
```
Comment (ColinKrist) [inline zoo_frontend/src/getPageContext.js]:
```suggestion
```

File: zoo_frontend/pages/home/home.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+// import Paper from '@material-ui/core/Paper';
```
Comment (ColinKrist) [inline zoo_frontend/pages/home/home.jsx]:
```suggestion
```

File: zoo_frontend/pages/home/home.jsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                <h3>2/11/2019</h3>
+              </div>
+              <Button variant="contained" className={classes.button} color="secondary">
+                Edit Diets
+              </Button>
+              <Button variant="contained" className={classes.button} color="secondary">
+                Edit Foods
+              </Button>
+            </div>
+            <VirtualTable />
```
Comment (ColinKrist) [inline zoo_frontend/pages/home/home.jsx]: Send in data as props here, I'd rather have a completed VirtualTable component than have to make further modifications to it in another PR. I'm not sure how big of an ask this is, but it is desired for sure.

File: zoo_frontend/components/VirtualTable.jsx [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+}
+
+function ReactVirtualizedTable() {
+  return (
+    <Paper style={{ height: 500, width: '100%' }}>
+      <WrappedVirtualizedTable
+        rowCount={rows.length}
+        rowGetter={({ index }) => rows[index]}
+        onRowClick={event => console.log(event)}
+        columns={[
```
Comment (ColinKrist) [inline zoo_frontend/components/VirtualTable.jsx]: columns should come in as props, rowCount and rowGetter can be figured out right above this return based off the props send in @ line 191.

**Review by ColinKrist — CHANGES_REQUESTED** [review]
VirtualizedTable component needs to have everything moved into props, so we don't have to touch this component again and can re-use it properly.

Other comments mostly for removing dead code.

File: zoo_frontend/pages/home/home.jsx (hunk 3 above)
Comment (buffginger) [reply zoo_frontend/pages/home/home.jsx]: I was not able to get it to work last night. If we want it for the demo I suggest we just merge what I have now. I'm sure it's a really dumb thing I'm missing and will be an simple fix.

**Review by ColinKrist — APPROVED** [review]
